src/vs/workbench/api/common/positron/extHost.positron.api.impl.ts

```diff
@@ -66,6 +67,7 @@ export function createPositronApiFactoryAndRegisterActors(accessor: ServicesAcce
 		rpcProtocol.getRaw(ExtHostContext.ExtHostLanguageFeatures);
 	const extHostEditors: ExtHostEditors = rpcProtocol.getRaw(ExtHostContext.ExtHostEditors);
 	const extHostDocuments: ExtHostDocuments = rpcProtocol.getRaw(ExtHostContext.ExtHostDocuments);
+	const extHostQuickOpen = rpcProtocol.set(ExtHostPositronContext.ExtHostQuickOpen, createExtHostQuickOpen(rpcProtocol, extHostWorkspace, extHostCommands));
```

Comment:
The way `ExtHostQuickOpen` is currently handled in VS Code is different from some of its brethren. There is no `IExtHostQuickOpen` and no support for doing `new ExtHostQuickOpen()` so I went ahead and got this working the same way they do it here:

https://github.com/posit-dev/positron/blob/48410356f157762cb5b5e6496b65ec18aba2f27a/src/vs/workbench/api/common/extHost.api.impl.ts#L198

Note the differences between this line and the ones above/below.